backend/controllers/Events.js

```diff
@@ -0,0 +1,80 @@
+const express = require('express')
+const bodyParser = require('body-parser')
+const cors = require('cors')
+const eventRouter = express.Router()
+
+var mongodb = require('mongodb');
+var Event = require('../models/Event');
+
+eventRouter.use(cors())
+eventRouter.use(bodyParser.json());
+eventRouter.use(bodyParser.urlencoded({
+  extended: true
+}));
+
+eventRouter.get('/list', function(req, res) {
+  Event.find(function(err, events) {
+    if (err) {
+      res.json(err);
```

Comment:
Should include a different response status if there's an error (good practice).  Refer to [this article](https://expressjs.com/en/guide/error-handling.html).  i.e. `res.status(500).send(err.message)` or `res.status(500).json(err')`

Reply:
Fixed error responses to show status